CIP5/README.md

```diff
@@ -48,6 +48,9 @@ We define the following set of common prefixes with their corresponding semantic
 | `vrf_vk`     | VRF verification key                          |
 | `vrf_vkh`    | VRF verification key hash                     |
 | `vrf_sk`     | VRF signing key                               |
+| `script`     | Script hash                                   |
+| `script_vkh` | Verification key hash of used in script       |
```

Comment:
*Verification key hash of key used in script?

Reply:
fixed in https://github.com/cardano-foundation/CIPs/pull/31/commits/6843b7b3e1ac401ba90f4bbec776465f45513aff